src/MizarData/include/MizarData/MizarData.h

```diff
@@ -28,7 +30,10 @@ class MizarData : public orbit_capture_client::AbstractCaptureListener<MizarData
  public:
   virtual ~MizarData() = default;
 
-  [[nodiscard]] std::optional<std::string> GetFunctionNameFromAddress(
+  [[nodiscard]] absl::flat_hash_map<uint64_t, std::string> AllAddressToName() const;
+
+  // virtual for testing purposes
+  [[nodiscard]] virtual std::optional<std::string> GetFunctionNameFromAddress(
```

Comment:
virtual for testing? why?

Reply:
I override it in testing, drastically simplifying the set up and making the unit-test actually _unit_.

Reply:
The `virtual` keyword is redundant if there is the `override`. I sent you a message in chat